src/status_im/ui/screens/contacts/subs.cljs

```diff
@@ -4,13 +4,11 @@
             [clojure.string :as str]
             [status-im.bots.constants :as bots-constants]))
 
-(reg-sub
-  :current-contact
+(reg-sub :current-contact
```

Comment:
why?

Reply:
more consistent derived subscription are always formatted like this, also makes it easier to find-grep